docs/dev/benchmarking.md

```diff
@@ -0,0 +1,77 @@
+# Benchmarking
+
+This document gives an overview of the benchmarking infrastructure of Concrete.
+
+## Concrete Python
+
+Concrete Python uses [progress-tracker-python](https://github.com/zama-ai/progress-tracker-python) to do benchmarks. Please refer to its [README](https://github.com/zama-ai/progress-tracker-python/blob/main/README.md) to learn how it works.
+
+### How to run all benchmarks?
+
+Use a makefile target:
```

Comment:
it's minor but isn't it "Use the makefile target"?

Reply:
Agreed, let me change it.